app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="container mb-3 p-2">
+  <div class="d-flex align-items-center justify-content-between">
+    <h1>Recipes</h1>
+    <%= link_to "New Recipe", new_recipe_path, class: 'btn btn-secondary' %>
+</div>
+
+  <div>
+    <% @recipes.each do |recipe| %>
+      <%= link_to recipe_path(recipe), class: "text-decoration-none text-dark" do %>
+          <%= render recipe %>
+      <% end %>
+    <% end %>
+  </div>
+
```

Comment:
- [ ] Kindly make sure a logged-in user, can add and delete their own recipes, currently, I see this error when I try to add a recipe, please check the requirements and ensure that you implement the functionalities  

![Screenshot_5](https://user-images.githubusercontent.com/69638013/233794085-c91286bf-f4fe-4a07-a63b-c4c11ec53cb4.png)

Reply:
@prolajumokeoni  Kindly fill the user input with the user id not the name .

Reply:
Hi @houdamzari thanks for pointing this out, please ensure that after successfully adding a  recipe only the owner of the recipe can delete it 👍🏿

Reply:
You did a great job working on this project! 🚀 However, there's still room to improve! I think what the previous reviewer meant was that the recipe should be created on behalf of the currently logged-in user, take in mind that regular users don't have access to the user's id as it is generated by the system.

- [ ] Please consider removing the User field and, instead, use the `current_user` method provided by devise to create a recipe on behalf of the current user.